docker-app/qfieldcloud/core/admin.py

```diff
@@ -49,13 +51,25 @@
     User,
     UserAccount,
 )
+from qfieldcloud.core.paginators import LargeTablePaginator
 from qfieldcloud.core.templatetags.filters import filesizeformat10
 from qfieldcloud.core.utils2 import jobs
 from rest_framework.authtoken.models import TokenProxy
 
+admin.site.unregister(LogEntry)
+
+
 Invitation = get_invitation_model()
 
 
+class ModelAdminEstimateCountMixin:
+    # Avoid repetitive counting large list views.
```

Comment:
As on production all lists are quite long one could consider adding the mixin to all Tables (TODO)?